crates/astria-composer/src/searcher/mod.rs

```diff
@@ -76,7 +75,11 @@ impl Searcher {
     /// Errors are returned in the following scenarios:
     /// + failed to connect to the eth RPC server;
     /// + failed to construct a sequencer clinet
-    pub(super) fn from_config(cfg: &Config) -> eyre::Result<Self> {
+    pub(super) fn from_config(
+        cfg: &Config,
+        serialized_rollup_transactions_tx: mpsc::Sender<SequenceAction>,
+        serialized_rollup_transactions_rx: mpsc::Receiver<SequenceAction>,
+    ) -> eyre::Result<Self> {
```

Comment:
In conductor we have a similar construction, but where the constructor returns a tuple `(Executor, ExecutorHandle)`. Can you do the same thing here?